core-processor/src/ext.rs

```diff
@@ -737,7 +737,11 @@ impl Externalities for Ext {
     }
 
     fn debug(&self, data: &str) -> Result<(), Self::UnrecoverableError> {
-        log::debug!(target: "gwasm", "DEBUG: {}", data);
+        let program_id = self.program_id()?;
+        let message_id = self.message_id()?;
+
+        log::debug!(target: "gwasm", "DEBUG: [handle({message_id})] {program_id}: {data}");
```

Comment:
Use precision to write ids shortly

Reply:
Should be resolved now. I set the precision to 2.